### PR TITLE
Fixes for local variables usage in SRC and ASRC components

### PR DESCRIPTION
**Pull request by singalsu**
This pull request contains cleanup for SRC component and avoiding local variables shadowing of function in ASRC by renaming better the functions in format.h header. Due to function rename DAI component is edited. The functionality of components is not changed.

### CODE REVIEW

File: src/audio/asrc/asrc.c [hunk 1]
```diff
@@ -535,9 +535,9 @@ static int asrc_prepare(struct comp_dev *dev)
 	struct comp_buffer *sourceb;
 	uint32_t source_period_bytes;
 	uint32_t sink_period_bytes;
-	int sample_bytes;
-	int sample_bits;
-	int frame_bytes;
+	int sbytes;
```
Comment (paulstelian97) [inline src/audio/asrc/asrc.c]: I don't actually like the change to less-descriptive names. If it's for alignment, I would rather you do the change the other way, the sof/audio/format.h functions themselves being changed to be more descriptive if reasonable.

Comment (singalsu) [reply src/audio/asrc/asrc.c]: Yes that's a good argument. Maybe prefixing the two functions with "get_" is better.

Comment (singalsu) [reply src/audio/asrc/asrc.c]: Done, thanks for suggestion!

File: src/audio/dai.c [hunk 2]
```diff
@@ -275,8 +275,8 @@ static int dai_playback_params(struct comp_dev *dev, uint32_t period_bytes,
 
 	/* set up DMA configuration */
 	config->direction = DMA_DIR_MEM_TO_DEV;
-	config->src_width = sample_bytes(dd->frame_fmt);
-	config->dest_width = sample_bytes(dd->frame_fmt);
+	config->src_width = get_sample_bytes(dd->frame_fmt);
+	config->dest_width = config->src_width;
 	config->cyclic = 1;
```
Comment (dbaluta) [inline src/audio/dai.c]: This is fine now, but please do ONE kind of change in a single patch. I would have added:

```
config->src_width = get_sample_bytes(dd->frame_fmt);
config->dest_width =  get_sample_bytes(dd->frame_fmt); 
```
And then later if you want:

```
 get_sample_bytes(dd->frame_fmt);
```

this is a small change and we can take it as it is. But please keep this in mind for next patches.

**Pull request merged**